includes/library/zencart/listingBox/src/Build.php

```diff
@@ -0,0 +1,134 @@
+<?php
+/**
+ * Class Build
+ *
+ * @copyright Copyright 2003-2015 Zen Cart Development Team
+ * @license http://www.zen-cart.com/license/2_0.txt GNU Public License V2.0
+ * @version $Id: $
+ */
+namespace ZenCart\ListingBox;
+use Aura\Di\Container;
+use Aura\Di\Factory;
+/**
+ * Class Build
+ * @package ZenCart\ListingBox
+ */
+class Build
+{
+    /**
+     * @param $zcDiContainer
+     * @param $listingBox
+     * @throws \Aura\Di\Exception\ContainerLocked
+     * @throws \Aura\Di\Exception\ServiceNotObject
+     * @throws \Exception
+     */
+    public function __construct($zcDiContainer, $listingBox)
```

Comment:
Never inject the global container.  Use a [factory](http://framework.zend.com/manual/current/en/in-depth-guide/services-and-servicemanager.html#writing-a-factory-class) instead.